scripts/09-history-migration-to-mongo.py

```diff
@@ -0,0 +1,455 @@
+                 for node in config["metadata"]["nodes"]]
+    logger.info('Connecting to meta nodes: {0}'.format(config["metadata"]["nodes"]))
+    meta_wait_timeout = config['metadata'].get('wait_timeout', 5)
+
+    try:
+        meta_node.add_remotes(addresses)
+    except Exception as e:
+        logger.error('Failed to connect to any elliptics meta storage node: {0}'.format(
+            e))
+        raise ValueError('Failed to connect to any elliptics storage META node')
+
+    meta_session = elliptics.Session(meta_node)
+    meta_session.set_timeout(meta_wait_timeout)
+    meta_session.add_groups(list(config["metadata"]["groups"]))
+
+    time.sleep(5)
+
+    return meta_session
+
+
+def get_mongo_client():
+    if not config.get('metadata', {}).get('url', ''):
+        raise ValueError('Mongo db url is not set')
+    return pymongo.mongo_replica_set_client.MongoReplicaSetClient(config['metadata']['url'])
+
+
+def create_collection(db, coll_name):
+    try:
+        db.create_collection(coll_name)
+    except pymongo.errors.CollectionInvalid:
+        raise RuntimeError('Collection coll_name already exists')
+
+
+def create_indexes(coll):
+    coll.ensure_index([
+        ('group_id', pymongo.ASCENDING)
+    ])
+    coll.ensure_index(
+        [
+            ('nodes.set.hostname', pymongo.ASCENDING),
+            ('nodes.set.port', pymongo.ASCENDING),
+            ('nodes.set.family', pymongo.ASCENDING),
+            ('nodes.set.backend_id', pymongo.ASCENDING),
+            ('nodes.set.path', pymongo.ASCENDING),
+        ],
+        name='hostname_1.port_1.family_1.backend_id_1.path_1'
+    )
+    coll.ensure_index([
+        ('nodes.timestamp', pymongo.ASCENDING),
+        ('nodes.type', pymongo.ASCENDING),
+    ])
+    coll.ensure_index([
+        ('couples.timestamp', pymongo.ASCENDING),
+        ('couples.type', pymongo.ASCENDING),
+    ])
+
+
+class GroupHistory(object):
+
+    GROUP_ID = 'group_id'
+    COUPLES = 'couples'
+    NODES = 'nodes'
+
+    PRIMARY_ID_KEY = GROUP_ID
+
+    PARAMS = (GROUP_ID, COUPLES, NODES)
+
+    def __init__(self, **init_params):
+        super(GroupHistory, self).__init__()
+
+        self.group_id = None
+        self.couples = []
+        self.nodes = []
+
+        for param in self.PARAMS:
+            if param in init_params:
+                if param == self.NODES:
+                    nodes_sets = []
+                    for init_nodes_set in init_params[param]:
+                        nodes_sets.append(GroupNodeBackendsSetRecord(**init_nodes_set))
+                    self.nodes = nodes_sets
+                elif param == self.COUPLES:
+                    couples = []
+                    for init_couples in init_params[param]:
+                        couples.append(GroupCoupleRecord(**init_couples))
+                    self.couples = couples
+                else:
+                    setattr(self, param, init_params[param])
```

Comment:
Changed it according to recent changes in history.py:
https://github.com/yandex/mastermind/commit/f10864346a596fceb2e45d3129125e1a0eabb65b#diff-506728a8428c50b7b64bccf1f8f354d2L102